portal/src/main/webapp/WEB-INF/jsp/mutation_details.jsp

```diff
@@ -128,7 +128,16 @@
                         annotation: annotationCol.render
                     },
                     columnVisibility: {
-                        annotation: 'visible'
+                        annotation: 'visible',
+                        "tumorType": function() {
```

Comment:
this function overrides the library default. if we want to keep the default behavior intact when `isVirtualStudy === false`, we still need to do something like this:

```JavaScript
"tumorType": function (util, gene) {
    if (isVirtualStudy) {
        return "visible";
    }
    else if (util.distinctTumorTypeCount(gene) > 1) {
        return "visible";
    }
    else {
        return "hidden";
    }
}
```